lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -240,6 +240,7 @@ export class PublicClientApplication extends ClientApplication implements IPubli
                 return this.acquireTokenByRefreshToken(silentRequest, requestWithCLP).catch((refreshTokenError: AuthError) => {
                     const isServerError = refreshTokenError instanceof ServerError;
                     const isInteractionRequiredError = refreshTokenError instanceof InteractionRequiredAuthError;
+                    const attemptSsoRTNotFound = (refreshTokenError.errorCode === BrowserConstants.NO_TOKENS_FOUND_ERROR);
```

Comment:
Should we name it just `rtNotFound` instead? Application logic that handles this scenario (refreshTokenError) is defined outside of this function.

Reply:
Makes sense. Done.